src/halmos/cheatcodes.py

```diff
@@ -461,15 +482,61 @@ def create_bytes32(ex, arg, **kwargs):
 
 
 def create_address(ex, arg, **kwargs):
-    name = name_of(extract_string_argument(arg, 0))
+    try:
+        name = name_of(extract_string_argument(arg, 0))
+    except (IndexError, NotConcreteError):
+        name = "vmRandomAddress"
     return ByteVec(uint256(create_generic(ex, 160, name, "address")))
 
 
 def create_bool(ex, arg, **kwargs):
-    name = name_of(extract_string_argument(arg, 0))
+    try:
+        name = name_of(extract_string_argument(arg, 0))
+    except (IndexError, NotConcreteError):
+        name = "vmRandomBool"
     return ByteVec(uint256(create_generic(ex, 1, name, "bool")))
 
 
+def create_uint256_min_max(ex, arg, **kwargs):
+    symbolic_value = create_generic(ex, 256, "vmRandomUint_min_max", "uint256")
+
+    min_value = uint256(extract_bytes(arg, 4 + 32 * 1, 32))
+    max_value = uint256(extract_bytes(arg, 4 + 32 * 2, 32))
+
+    min_value = uint256(min_value).as_z3()
+    max_value = uint256(max_value).as_z3()
+
+    if eq(simplify(min_value), simplify(max_value)):
```

Comment:
this simplifies both but then discards the result, it would be better to save the simplified values